### DISCUSSION TITLE
Github tools and apps on the Github Marketplace?

### DISCUSSION THREAD

**Discussion by jibesh-bhattacharjee** [community/community#21538] [New to GitHub]
<p>Can anyone please suggest some cool tools or Apps available on the Github Marketplace?</p>

**Comment by JulienBreux** [comment, marked as answer]
<p>Hi <a class="mention" href="/jibesh-bhattacharjee">@jibesh-bhattacharjee</a>,</p>
<p>Code Climate is a very nice tool, <a href="https://github.com/marketplace/code-climate" rel="nofollow noopener">here</a> :wink:</p>

**Comment by jibesh-bhattacharjee** [comment]
<p>Cool. Thanks <a class="mention" href="/julienbreux">@julienbreux</a></p>
<p>Can you please share a bit about how Code Climate has been useful to you?</p>
